client/contivModelClient.go

```diff
@@ -80,7 +136,16 @@ type Link struct {
 
 func httpGet(url string, jdata interface{}) error {
 
-	r, err := http.Get(url)
+	req, err := http.NewRequest("GET", url, nil)
+	if err != nil {
+		return err
+	}
+
+	if err := addAuthTokenHeader(req); err != nil {
```

Comment:
See line 95.  Non-HTTPS requests return early and don't get the header added (for security reasons)